app/webpacker/components/Tickets/TicketComments.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { Button, Comment, Header } from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import { DateTime } from 'luxon';
+import TicketCommentCreate from './TicketCommentCreate';
+import getComments from './api/getComments';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import '../../stylesheets/semantic/components/comment.min.css';
+
+export default function TicketComments({ ticketId, currentStakeholder }) {
+  const [createComment, setCreateComment] = useState(false);
+
+  const {
+    data: { comments } = { comments: [] }, isFetching, isError, refetch,
```

Comment:
If the `comments` are really all that you care about, then you should pass a `select` option: https://tanstack.com/query/latest/docs/framework/react/guides/render-optimizations#select